packages/playwright-core/src/server/android/android.ts

```diff
@@ -238,20 +238,19 @@ export class AndroidDevice extends SdkObject {
   async launchBrowser(pkg: string = 'com.android.chrome', options: types.BrowserContextOptions): Promise<BrowserContext> {
     debug('pw:android')('Force-stopping', pkg);
     await this._backend.runCommand(`shell:am force-stop ${pkg}`);
-
-    const socketName = 'playwright-' + createGuid();
+    const socketName = isUnderTest() ? 'chrome_devtools_remote' : ('playwright-' + createGuid());
```

Comment:
'chrome_devtools_remote' will clash with Chrome, let's get a unique prefix for Playwright test.